### ISSUE TITLE
Support descriptions of modes

### ISSUE THREAD

**Issue by janiemi** [spraakbanken/korp-frontend#262]
Could the modes have descriptions of some kind? Even though mode names are often self-descriptive, not all of them are necessarily such to everyone. (At least I didn’t know what “Kubhist” in Språkbanken’s Korp referred to until I actually looked at it.) And even if the name were descriptive, a separate description could contain some additional information.

The description could perhaps be shown as a tooltip when hovering over the name of the mode. However, would it also make sense to display the description of the active mode also be somewhere, perhaps in the corpus chooser? In that case, the user should perhaps be allowed to hide the description so that it wouldn’t take up (much) space.

The mode descriptions could also be thought of as pieces of documentation in the mode files of the configuration.

**Comment by majsan** [comment]
I agree that modes should have descriptions and tooltips seems like an easy way to do this. With the new format for translation in the mode configuration file:
```
description:
  eng: Desc. in English
  swe: Beskrivning på svenska
```

**Comment by majsan** [comment]
This was partly fixed in 0fd20111b, the description will be shown in the empty space where results appear. It doesn't solve the issue with getting the description before going to the mode though.

**Comment by arildm** [comment]
Since 525f00e2 there is also `mode_description`. If given, it is appended to `description` on the frontpage, with the mode name as a subheading. See https://korpdev.spraakbanken.gu.se:9111/?mode=kubhist for example.

I agree that showing one-line summaries in the mode chooser would be useful too.
